tests/test_graphql.py

```diff
@@ -201,6 +201,33 @@ def test_product_query(admin_client, product):
         'priceRange']['stop']['gross']['amount']
 
 
+def test_product_with_collections(admin_client, product, collection):
+    query = '''
+        query getProduct($productID: ID!) {
+            product(id: $productID) {
+                collections {
```

Comment:
Please test if pagination works (a detail, but we've seen today that we should do it)